mne/preprocessing/tests/test_maxwell.py

```diff
@@ -253,4 +264,66 @@ def test_spatiotemporal_maxwell():
     # Degenerate cases
     assert_raises(ValueError, maxwell_filter, raw, st_dur=10., st_corr=0.)
 
+
+@testing.requires_testing_data
+def test_maxwell_filter_fine_calibration():
+    """Test fine calibration feature of Maxwell filter"""
+
+    # Note: Elekta MaxFilter uses single precision, so expect filtered results
+    # to differ slightly.
+
+    # Load testing data (raw, SSS std origin, SSS non-standard origin)
+    with warnings.catch_warnings(record=True):  # maxshield
+        raw = Raw(raw_fname, preload=False, proj=False,
+                  allow_maxshield=True).crop(0., 1., False)
+    raw.preload_data()
+    raw.pick_types(meg=True, eeg=False)
+    with warnings.catch_warnings(record=True):  # maxshield, naming
+        sss_fine_cal = Raw(sss_fine_cal_fname, preload=True, proj=False,
+                           allow_maxshield=True)
+
+    # Create coils
+    all_coils, _, _, meg_info = _prep_meg_channels(raw.info, ignore_ref=True,
+                                                   elekta_defs=True)
+    picks = [raw.info['ch_names'].index(ch) for ch in [coil['chname']
+                                                       for coil in all_coils]]
+
+    # Test 1D SSS fine calibration
+    raw_sss = maxwell_filter(raw, origin=[0., 0., 40.], int_order=int_order,
+                             ext_order=ext_order,
```

Comment:
Only pass args that are not defaults. It helps ensure we don't change defaults in some bad way. So you should only need to pass `(raw, fine_cal_fname=fine_cal_fname)`.